src/ts/src/Landing.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+// import cx from "classnames";
```

Comment:
intended?

Reply:
No, not yet anyways. As for `watchOptions`, that does not solve the issue since those two lines specifically tell the plugin to watch `node_modules`.

It seems to be behaving today so I can't reproduce the stack trace :/ perhaps all the heavy dev I was doing last week confused it? I'm not sure. But I can undo these changes.